.github/CODEOWNERS

```diff
@@ -162,9 +160,9 @@
 /src/pages/gen1/\[platform\]/prev/build-a-backend/utilities/ @josefaidt
 
 #Deploy and Host 
-/src/pages/gen1/\[platform\]/deploy-and-host/ @mauerbac
-/src/pages/gen1/\[platform\]/prev/deploy-and-host/ @mauerbac
-/src/pages/\[platform\]/deploy-and-host/ @mauerbac
+/src/pages/gen1/\[platform\]/deploy-and-host/ @mauerbac @josefaidt
+/src/pages/gen1/\[platform\]/prev/deploy-and-host/ @mauerbac @josefaidt
+/src/pages/\[platform\]/deploy-and-host/ @mauerbac @josefaidt
 
 #Docs Engineering
```

Comment:
Don't these need to be updated as well? ([reference](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners))
> Order is important; the last matching pattern takes the most precedence

Reply:
I _think_ these ones are okay since they aren't targeting the content at `src/pages/[platform]` and `src/pages/gen1`

Reply:
`/src/pages/**/index.tsx @aws-amplify/documentation-team` would trigger for any content change, no? That looks like the only problematic one though.

Reply:
only if the page is a react component, which there are a few but not too many. We're more concerned about mdx pages `src/pages/**/index.mdx`

Reply:
Ok, makes sense. Thanks!